packages/jest-fake-timers/src/legacyFakeTimers.ts

```diff
@@ -471,7 +471,10 @@ export default class FakeTimers<TimerRef> {
   }
 
   private _fakeRequestAnimationFrame(callback: Callback) {
-    return this._fakeSetTimeout(callback, 1000 / 60);
+    return this._fakeSetTimeout(() => {
+      // TODO: Use performance.now() once it's mocked
+      callback(this._now);
```

Comment:
`_now` is ms, right? The callback should receive microseconds?

Reply:
The callback should receive a [`DOMHighResTimestamp`](https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp) which is in milliseconds. We just don't have any precision which is safe since browsers may use an imprecise timestamp as well

Reply:
ah, cool :+1: